contracts/ColonyStorage.sol

```diff
@@ -150,11 +145,6 @@ contract ColonyStorage is CommonStorage, ColonyDataTypes, DSMath {
 
     // Prevent people moving funds from the pot designated to paying out token holders
     require(_fromPot > 0, "colony-funding-cannot-move-funds-from-rewards-pot");
-
-    // Preventing sending from non-existent funding pots is not strictly necessary (if a pot doesn't exist, it can't have any funds if we
-    // prevent sending to nonexistent funding pots) but doing this check explicitly gives us the error message for clients.
-    require(_fromPot <= fundingPotCount, "colony-funding-from-nonexistent-pot"); // Only allow sending from created pots
-    require(_toPot <= fundingPotCount, "colony-funding-nonexistent-pot"); // Only allow sending to created funding pots
```

Comment:
Why were these removed?

Reply:
That logic was moved to https://github.com/JoinColony/colonyNetwork/pull/553/files#diff-8920cdf7ab56c438c907df9bd37d2931R471